packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -49,13 +49,15 @@ export type StripePaymentSubmitResult = {
   name: string;
   card: StripeCardElement | null;
   idempotencyKey: string;
+  promoCode: string | undefined;
```

Comment:
nit: a little inconsistent with the `promoCode?: string;` from above.

Reply:
updated